models/discordactions.js

```diff
@@ -87,9 +90,37 @@ const addGroupRoleToMember = async (roleData) => {
   }
 };
 
+/**
+ *
+ * @param userDiscordId { String }: DiscordId of the user
+ * @returns {Promise<String>}
+ */
+const updateDiscordImageForVerification = async (userDiscordId) => {
+  try {
+    const discordAvatarUrl = await generateDiscordProfileImageUrl(userDiscordId);
+    const verificationDataSnapshot = await photoVerificationModel
+      .where("discordId", "==", userDiscordId)
+      .limit(1)
+      .get();
+    const unverifiedUserDiscordImage = {
+      discord: { url: discordAvatarUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
+    };
+    if (verificationDataSnapshot.empty) {
+      throw new Error("No user verification record found");
+    }
+    const documentRef = verificationDataSnapshot.docs[0].ref;
+    await documentRef.update(unverifiedUserDiscordImage);
```

Comment:
as we discussed in call yesterday if we update the image url we will miss out their old image url.

Reply:
The change was actually suggested in the last minute, it would require me to re-write the whole code. So, went with it. Will change it creating a new issue ticket.